src/main/scad/yavuz-demirhan/Raya1.scad

```diff
@@ -0,0 +1,66 @@
+include <puzzlecad2.scad>
+
+box_puzzle_scale = 15;
+box_puzzle_border = 6;
+$burr_inset = 0.16; 
```

Comment:
Any reason for going with 0.16 mm inset rather than 0.125 (as in other packing puzzles)? Was the smaller inset too tight on your prints?

(I'm not necessarily objecting to a larger inset, but we should be consistent... i.e. use the same value here as in Mushkila and the other packing puzzles, unless there's a good reason they should be different)

Reply:
Hi Aaron, on my printer 0.125 seems to be too tight for packing puzzles but I agree it should be standard. I will adjust to standard.

Reply:
hmm. I have gotten reports from other users that default tolerances are too tight.. it seems there is a lot of variation in tolerances by printer. I am thinking it might make sense to distribute two versions of every puzzle with different tolerances (as different STLs, built from the same .scad)...